src/js/popup.js

```diff
@@ -110,6 +110,17 @@ function init() {
   var overlay = $('#overlay');
   $("#error").on("click", function() {
     overlay.toggleClass('active');
+
+    $('#error_input').bind('input propertychange', function() {
```

Comment:
Do we rebind over and over, once per error button click? We should move the binding then to do it only once.